frontend/src/scss/Core.scss

```diff
@@ -126,13 +126,31 @@ form {
   }
 }
 
+.cta-buttons {
+  button {
+    display: flex;
+    justify-content: center;
+    padding: 0 !important;
+
+    span {
+      display: flex;
+      align-items: center;
+      line-height: 30px;
+    }
+  }
+}
+
```

Comment:
Wasn't this going to be for all buttons?

Reply:
Yeah. Fixed it in the other PR. Gonna copy the code in the branch and put it in here.